### PR TITLE
Feature/configure percy

### PR DESCRIPTION
**Pull request by nowak-pawel-ds**

### CODE REVIEW

File: .github/workflows/build-checks.yml [hunk 1]
```diff
@@ -39,7 +44,15 @@ jobs:
           mkdir -p ~/.m2/repository
           cp .github/settings.xml ~/.m2/settings.xml
           echo "CODEARTIFACT_AUTH_TOKEN=${CODEARTIFACT_AUTH_TOKEN}" >> $GITHUB_ENV
-      - name: Build and test with Maven
+      - name: Build and test cypress with Maven
```
Comment (marcell-pal-ds) [inline .github/workflows/build-checks.yml]: Do we test Cypress with Maven?

**Comment by marcell-pal-ds** [comment]
I think you should rebase this PR to `origin/main` or squash the commits, because there are lots of left-over commits in this PR.

File: .github/workflows/build-checks.yml (hunk 1 above)
Comment (nowak-pawel-ds) [reply .github/workflows/build-checks.yml]: it is inside maven build as execution step

Comment (nowak-pawel-ds) [reply .github/workflows/build-checks.yml]: but right we are not testing Cypress but with Cypress

**Comment by nowak-pawel-ds** [comment]
@marcell-pal-ds  I will squash merge so only merge commit will be visible. For some reason merge in github (I merged origin/main) still puts it in commits list

**Review by marcell-pal-ds — APPROVED** [review]

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**